From 574ea07da1609eba4e608cb8fddbe6e98b88b091 Mon Sep 17 00:00:00 2001
From: Mike Perry <mikeperry-git@torproject.org>
Date: Sun, 31 Mar 2013 22:48:00 -0700
Subject: [PATCH 07/41] Disable SSL Session ID tracking.

We can't easily bind SSL Session ID tracking to url bar domain,
so we have to disable them to satisfy
https://www.torproject.org/projects/torbrowser/design/#identifier-linkability.
---
 security/nss/lib/ssl/sslsock.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/security/nss/lib/ssl/sslsock.c b/security/nss/lib/ssl/sslsock.c
index 4f4b034..6ce5d11 100644
--- a/security/nss/lib/ssl/sslsock.c
+++ b/security/nss/lib/ssl/sslsock.c
@@ -141,7 +141,7 @@ static sslOptions ssl_defaults = {
     PR_FALSE,	/* enableSSL2         */ /* now defaults to off in NSS 3.13 */
     PR_FALSE,	/* unusedBit9         */
     PR_FALSE, 	/* unusedBit10        */
-    PR_FALSE,	/* noCache            */
+    PR_TRUE,	/* noCache            */
     PR_FALSE,	/* fdx                */
     PR_FALSE,	/* v2CompatibleHello  */ /* now defaults to off in NSS 3.13 */
     PR_TRUE,	/* detectRollBack     */
-- 
1.8.1.2

